src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -28,22 +26,41 @@ partial class ApplicationEngine
         /// </summary>
         public static readonly InteropDescriptor System_Contract_CreateStandardAccount = Register("System.Contract.CreateStandardAccount", nameof(CreateStandardAccount), 0_00010000, CallFlags.None, true);
 
-        protected internal void CreateContract(byte[] script, byte[] manifest)
+        private UInt160 ContractHash(NefFile nef)
+        {
+            if (!(ScriptContainer is Transaction tx))
+                throw new InvalidOperationException();
+
+            using var script = new ScriptBuilder();
+            script.Emit(OpCode.ABORT);
+            script.EmitPush(tx.Sender);
+            script.EmitPush(nef.ScriptHash);
```

Comment:
`Script` or `ScriptHash`? Maybe we can remove `ScriptHash` from nef file.

Reply:
If we remove `ScriptHash` we should remove the `Checksum`. It's ok for you?

Reply:
Add Script into it is okay for me.

Reply:
Also remove `ScriptHash` from nef file?  
https://github.com/neo-project/neo/pull/2044/files#diff-65375a6e30513a2b43371cfa09b221084059d0d1465dc6e91a6ffbc327f948f7L42

Reply:
Done